langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -123,6 +153,10 @@ public isolated function min(int n, int... ns) returns int = @java:Method {
 # The first character may be `+` or `-`.
 # This is the inverse of function ``value:toString`` applied to an `int`.
 #
+# ```ballerina
+# int number = check int:fromString("56");
```

Comment:
Let's use a variable as the argument. This wouldn't usually be a way to use this function, would do `int number = 56;` instead.